src/themes/jetstream/color/base.json

```diff
@@ -0,0 +1,110 @@
+{
+  "color": {
+    "jetstream": {
```

Comment:
I have pretty much doing the same thing like in [Excursion theme's PR](https://github.com/AlaskaAirlines/AuroDesignTokens/pull/156), but originally, this was `color.base`.
If name changes for this are not needed, then I will update this with all of the references.

Reply:
We just released the first draft of the new excursion theme and made a number of changes in the process, you'll probably need to doublecheck that.  Let us know if you have any trouble mapping css values to the new tier 2 token names.